src/components/Vote.jsx

```diff
@@ -0,0 +1,30 @@
+import React, {useState } from 'react';
+import{updateArticleVote} from '../utils'
+
+export default function VoteArticle({article, setArticle}) {
+  const [voteChange, setVoteChange] = useState(0);
+
+  function handleVote(incVote) {
+    setArticle(article => ({ ...article, votes: article.votes + incVote }));
+    updateArticleVote(article.article_id, incVote)
+    // setVoteChange(currVoteChange => currVoteChange + incVote)
+    .then(() => setVoteChange(currVoteChange => currVoteChange + incVote))
+    .catch(error => console.error('Failed to update vote:', error));
```

Comment:
logging the error to console here isn't really enough feedback for our client. We would expect some sort of message in the client view indicating there had been an error.